biojava-structure/src/test/java/org/biojava/nbio/structure/io/cif/CifFileSupplierTest.java

```diff
@@ -0,0 +1,192 @@
+package org.biojava.nbio.structure.io.cif;
+
+import org.biojava.nbio.structure.AminoAcidImpl;
+import org.biojava.nbio.structure.Atom;
+import org.biojava.nbio.structure.AtomImpl;
+import org.biojava.nbio.structure.Chain;
+import org.biojava.nbio.structure.ChainImpl;
+import org.biojava.nbio.structure.Element;
+import org.biojava.nbio.structure.EntityInfo;
+import org.biojava.nbio.structure.Group;
+import org.biojava.nbio.structure.ResidueNumber;
+import org.biojava.nbio.structure.Structure;
+import org.biojava.nbio.structure.StructureImpl;
+import org.biojava.nbio.structure.StructureTools;
+import org.biojava.nbio.structure.io.CifFileReader;
+import org.biojava.nbio.structure.io.FileParsingParameters;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Arrays;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class CifFileSupplierTest {
```

Comment:
It'd be better to put this test under the integration test module. We'd like to have a clearer separation of unit tests and integration tests going forward: the structure module should have unit tests, and the integrationtest module should have integration tests.

Reply:
Seems like we had that discussion already. Just saw that the same test exists in the integrationtest module :) (https://github.com/biojava/biojava/blob/master/biojava-integrationtest/src/test/java/org/biojava/nbio/structure/test/io/cif/CifFileSupplierIntegrationTest.java). I deleted the redundant copy in biojava-structure.